src/ui/localization/translator.ts

```diff
@@ -0,0 +1,371 @@
+import en from "./locale/en.json";
+import es from "./locale/es.json";
+import it from "./locale/it.json";
+import ar from "./locale/ar.json";
+import ca from "./locale/ca.json";
+import zh_Hans from "./locale/zh_Hans.json";
+import zh_Hant from "./locale/zh_Hant.json";
+import hr from "./locale/hr.json";
+import cs from "./locale/cs.json";
+import da from "./locale/da.json";
+import nl from "./locale/nl.json";
+import fi from "./locale/fi.json";
+import fr from "./locale/fr.json";
+import de from "./locale/de.json";
+import el from "./locale/el.json";
+import he from "./locale/he.json";
+import hi from "./locale/hi.json";
+import hu from "./locale/hu.json";
+import id from "./locale/id.json";
+import ja from "./locale/ja.json";
+import ko from "./locale/ko.json";
+import ms from "./locale/ms.json";
+import no from "./locale/no.json";
+import pl from "./locale/pl.json";
+import pt from "./locale/pt.json";
+import ro from "./locale/ro.json";
+import ru from "./locale/ru.json";
+import sk from "./locale/sk.json";
+import sv from "./locale/sv.json";
+import th from "./locale/th.json";
+import tr from "./locale/tr.json";
+import uk from "./locale/uk.json";
+import vi from "./locale/vi.json";
+
+import type { PeriodUnit } from "../../helpers/duration-helper";
+
+export type emptyString = "";
```

Comment:
Hmm I wonder if this should be uppercased
```suggestion
export type EmptyString = "";
```

Reply:
Yep, type not const. I refactored it from const to type and forgot to uppercase it, thanks!